lib/update/cluster/phases/kubernetes.go

```diff
@@ -378,4 +379,84 @@ func removeKubeletPermissions(client *kubeapi.Clientset) error {
 	return nil
 }
 
+func getInfluxDBNodename(client *kubeapi.Clientset, logger log.FieldLogger) (string, error) {
+	var labels map[string]string
+	deployment, err := getInfluxDBDeployment(client)
+	if deployment.Spec.Selector != nil {
+		labels = deployment.Spec.Selector.MatchLabels
+	}
+	replicaSets, err := rigging.CollectReplicaSets(deployment.Namespace, labels, logger, client, func(ref metav1.OwnerReference) bool {
```

Comment:
Can't we just query the influxdb pod by its labels in the monitoring namespace? Should be easy enough, a single API call:

```
ubuntu@g5-cluster-node-1:~$ kubectl -nmonitoring get pods/influxdb-677c446f6f-k28g4 -oyaml
apiVersion: v1
kind: Pod
metadata:
  annotations:
    kubernetes.io/psp: privileged
    seccomp.security.alpha.kubernetes.io/pod: docker/default
  creationTimestamp: "2020-05-29T22:24:42Z"
  generateName: influxdb-677c446f6f-
  labels:
    app: monitoring
    component: influxdb
    pod-template-hash: 677c446f6f
  name: influxdb-677c446f6f-k28g4
  namespace: monitoring
...
```